MVP Module/Source/Feed Presentation/FeedErrorViewModel.swift

```diff
@@ -0,0 +1,11 @@
+//
+//  FeedErrorViewModel.swift
+//  MVP
+//
+//  Created by Haik Ampardjian on 21.08.2021.
+//  Copyright © 2021 Essential Developer. All rights reserved.
+//
+
+struct FeedErrorViewModel {
+	let errorMessage: String
```

Comment:
Suggestion: 
Since it is an **error** viewModel, the context is pretty clear and the property can be named simply `message`.
Also the message can be optional following my next comment on `FeedErrorView`
```suggestion
	let message: String?
```